extensions/jackson-avro/runtime/pom.xml

```diff
@@ -34,6 +34,7 @@
 
     <properties>
         <camel.quarkus.jvmSince>1.8.1</camel.quarkus.jvmSince>
+        <camel.quarkus.nativeSince>1.8.1</camel.quarkus.nativeSince>
```

Comment:
Nice that you have caught this :) An automatic check to ensure that these versions are not newer than the current snapshot would be handy.